styles.css

```diff
@@ -100,3 +117,160 @@ h2.title {
   justify-content: space-between;
   list-style-type: none;
 }
+
+.projects-section {
+  padding: 26px;
+  background: url("./images/cylinder_2_page_top_right.png") top right no-repeat;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 24px;
+}
+
+.project {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+  height: 474px;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+}
+
+.technologies {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  width: 100%;
+}
+
+.tech {
+  background: #ebf0ee;
+  padding: 8px 12px;
+  border-radius: 4px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.project-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.see-project-button {
+  background: #36b37f;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+.see-project-button:active {
+  background: #008552;
+}
+
+.see-project {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.project-description {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  height: 222px;
+  align-items: center;
+  background: #fff;
+}
```

Comment:
- [ ] **[OPTIONAL]** If we compare your card to the figma template, you will be able to notice that yours is lacking the `border-radius` property on the `project-description`. Can you do it for us?